egs2/chime7_task1/diar_asr1/local/database.yml

```diff
@@ -0,0 +1,49 @@
+Databases:
```

Comment:
Where and how is this file used?
It would be better to add one line comment in the header.